cg/server/api.py

```diff
@@ -67,7 +60,7 @@
 @BLUEPRINT.route("/submit_order/<order_type>", methods=["POST"])
 def submit_order(order_type):
     """Submit an order for samples."""
-    api = OrdersAPI(lims=lims, status=db, osticket=osticket)
```

Comment:
This endpoint is defined in `/server/endpoints/orders.py`, not in this file any longer. I think you missed that when you merged master!